client/app/scripts/components/terminal.js

```diff
@@ -243,7 +243,7 @@ class Terminal extends React.Component {
 
     const bcr = this.node.getBoundingClientRect();
     const minWidth = (this.state.characterWidth * 80) + (8 * 2);
-    openNewWindow(`${getApiPath()}/terminal.html#!/state/${paramString}`, bcr, minWidth);
+    openNewWindow(`${basePath(window.location.pathname)}/terminal.html#!/state/${paramString}`, bcr, minWidth);
```

Comment:
`getApiPath` was already using `basePath` internally.

Reply:
This will now try and open:

`/app/:instanceId/terminal.html` instead of `/api/app/:instanceId/terminal.html` e.g. the service-ui will now handle terminal.html rather than query service